src/rgw/rgw_rest_s3.cc

```diff
@@ -979,14 +973,11 @@ void RGWListBucket_ObjStore_S3::send_common_response()
     }
   }
   
-
-
 void RGWListBucket_ObjStore_S3::send_response()
 {
-   
-if(op_ret < 0) {
-    set_req_state_err(s, op_ret);
-  }
+  if (op_ret < 0) {
+      set_req_state_err(s, op_ret);
+    }
```

Comment:
lines 979 and 980 have 2 extra spaces of indent

Reply:
@cbodley This branch is not in my local machine. Updated via github editor.